installer/Translations/SimpChinese.wxl

```diff
@@ -2,48 +2,49 @@
 <WixLocalization Culture="zh-cn" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="LANG">2052</String>
 
+  <!-- By deluxghost -->
```

Comment:
I would remove this line, as we have a dedicated file (`AUTHORS`) for the contributors.